core/src/style/polylineStyle.cpp

```diff
@@ -15,8 +16,7 @@ void PolylineStyle::constructVertexLayout() {
     m_vertexLayout = std::shared_ptr<VertexLayout>(new VertexLayout({
         {"a_position", 3, GL_FLOAT, false, 0},
         {"a_texcoord", 2, GL_FLOAT, false, 0},
-        {"a_extrudeNormal", 2, GL_FLOAT, false, 0},
-        {"a_extrudeWidth", 1, GL_FLOAT, false, 0},
+        {"a_extrude", 4, GL_FLOAT, false, 0},
         {"a_color", 4, GL_UNSIGNED_BYTE, true, 0},
```

Comment:
extrudeNormal could be premultiplied with extrudeWidth to save 4 bytes per vertex I guess